Games/types/Mafia/roles/cards/Poisoner.js

```diff
@@ -1,5 +1,5 @@
 const Card = require("../../Card");
-const { PRIORITY_POISONER } = require("../../const/Priority");
+const { PRIORITY_EFFECT_GIVER_DEFAULT } = require("../../const/Priority");

```

Comment:
In the priorities file, you're deleting `const PRIORITY_POISONER = PRIORITY_KILL_DEFAULT`
Shouldn't the Poisoner priority be replaced by the kill default here?

Reply:
this action is not a killing action, it is giving people effect only

the poison effect has a priority of 0 (not stated, 0 is the default)